style.css

```diff
@@ -0,0 +1,642 @@
+  font-weight: 500;
+  color: #ec5242;
+}
+
+.p2-p3 {
+  font-size: 8px;
+  font-weight: 100;
+  color: #272a31;
+}
+
+.body-p2 {
+  width: 70%;
+}
+
+.img-tg {
+  width: 20%;
+}
+
+.img-tg img {
+  width: 100%;
+}
+
+.partners-section {
+  background-color: #272a31;
+  padding: 0;
+  margin: 0;
+}
+
+.h-ptns {
+  align-content: center;
+  color: #d3d3d3;
+  text-align: center;
+  margin: 0;
+}
+
+.ul-list {
+  display: flex;
+  list-style: none;
+  gap: 50px;
+  color: #d3d3d3;
+  justify-content: center;
+  text-align: center;
+  margin: 0;
+  padding: 10px;
+}
+
+.li-li {
+  padding: 5px;
+}
+
+.rights-section {
+  display: flex;
+  gap: 30px;
+  align-content: center;
+  background-color: #d3d3d3;
+  height: auto;
+  justify-content: center;
+}
+
+@media (min-width: 768px) {
+  .card-main {
+    padding: 20px 0;
+  }
+
+  .spks-main {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    justify-content: center;
+    width: 90%;
+    margin: 0 auto;
+    row-gap: 20px;
+  }
+}
+
+.about-section {
+  background: #d3d3d3;
+}
+
+.about-pr {
+  border: 1px solid #d3d3d3;
+  text-align: center;
+  background: #fff;
+  padding: 55px;
+  margin: 40px;
+}
+
+.about-address {
+  text-align: center;
+}
+
+.sub-about {
+  background: #fff;
+}
+
+.sub-abt-h {
+  font-size: 35px;
+  text-align: center;
+  color: #ec5242;
+}
+
+.sub-abt-h1 {
+  font-size: 25px;
+  text-align: center;
+}
+
+.about-pr-2 {
+  text-align: center;
+}
+
+.lg-abt {
+  text-align: center;
+}
+
+.la-abt {
+  color: #272a31;
+}
+
+.im-2 {
+  background-image: url(Images/im.jpg);
+  width: 369px;
+  height: 180px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  margin: 7px;
+}
+
+.h3-1 {
+  color: #fff;
+  text-align: center;
+}
+
+.p3-1 {
+  text-align: center;
+  color: #fff;
+}
+
+.im4-1 {
+  background-image: url(Images/imd.jpg);
+}
+
+.menu-2 {
+  cursor: pointer;
+  align-self: center;
+}
+
+.bar-2 {
+  display: block;
+  width: 35px;
+  margin: 4px auto;
+  background-color: black;
+  height: 4px;
+}
+
+.hmpage-top-bars {
+  display: none;
+  justify-content: flex-end;
+  gap: 10px;
+  background: #272a31;
+  background: black;
+  height: 4vh;
+  color: white;
+  align-items: center;
+  padding-right: 4%;
+}
+
+.hmpage-desktop {
+  display: none;
+  background: #fff;
+}
+
+.cards-2 {
+  display: flex;
+  justify-content: center;
+  gap: 5%;
+  flex-direction: column;
+}
+
+.lst-part {
+  display: flex;
+  justify-content: center;
+  background: #272a31;
+}
+
+.lst-logo {
+  display: flex;
+}
+
+.lst-name {
+  color: #fff;
+}
+
+.lst-pr {
+  color: #fff;
+}
+
+.lg-lst {
+  display: flex;
+  justify-content: center;
+}
+
+.log-lst {
+  padding: 10px;
+  font-weight: 700;
+  color: #fff;
+}
+
+@media (min-width: 768px) {
+  .cards-2 {
+    display: flex;
+    justify-content: center;
+    gap: 5%;
+    flex-direction: row;
+  }
+
+  .lst-part {
+    display: flex;
+    justify-content: center;
+    background: #fff;
+  }
```

Comment:
- [ ] Kindly notice that the footer of the about page should be dark as the Figma design.

Your Design | Figma Design
------------- | ----------------
![Capture](https://user-images.githubusercontent.com/33752735/204374384-806133ee-4541-409c-858b-b001e0daae66.PNG) | ![Capture1](https://user-images.githubusercontent.com/33752735/204374425-eadf8b70-322a-4664-98b4-ce492d5929be.PNG)